qa/tasks/vstart_runner.py

```diff
@@ -678,7 +679,7 @@ def is_blocked(self):
         return self.addr in output
 
 
-class LocalKernelMount(LocalCephFSMount, KernelMount):
+class LocalKernelMount(LocalCephFSMount, tasks.cephfs.kernel_mount.KernelMountBase):
```

Comment:
please expand in the commit message why this class renaming was all necessary (perhaps in a separate commit)

Reply:
Done